libavif: treat qcc as GCC.

CMake identifies qcc, a GCC driver, as "QCC"; take the GNU flag branch.

Upstream-Status: QNX-specific

diff --git a/CMakeLists.txt b/CMakeLists.txt
index baf5964..9806222 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -304,7 +304,8 @@ if(MSVC)
 elseif(CMAKE_C_COMPILER_ID MATCHES "Clang")
     message(STATUS "libavif: Enabling warnings for Clang")
     target_compile_options(avif_obj PUBLIC $<BUILD_INTERFACE:-Wall -Wextra -Wshorten-64-to-32>)
-elseif(CMAKE_C_COMPILER_ID MATCHES "GNU")
+elseif(CMAKE_C_COMPILER_ID MATCHES "GNU|QCC")
+    # QNX's qcc is a GCC driver, but CMake identifies it as "QCC".
     message(STATUS "libavif: Enabling warnings for GCC")
     target_compile_options(avif_obj PUBLIC $<BUILD_INTERFACE:-Wall -Wextra>)
 else()
